### ISSUE TITLE
Can't see OMEMO messages sent from Conversations

### ISSUE THREAD

**Issue by zzottel** [kalkin/gajim-omemo#4]
When I send a message from Gajim, I can see it in my account in Conversations (only unencrypted yet, see other issue).

The same is true for unencrypted messages sent in Conversations. If I send encrypted messages in Conversations, though, I can't see them in Gajim.

The server is Prosody 0.10 with mod_carbons active.

Gajim version is 0.16 on Lubuntu 15.04.

**Comment by kalkin** [comment]
Hi,

thank you for your report. You are stating in #3 : 

> when the other end writes an OMEMO-encrypted message, I can read it. 

But here you are telling me: 

> If I send encrypted messages in Conversations, though, I can't see them in Gajim.

No I'm officially confused :smile: Are you using in #3 something other as Conversation for sending OMEMO messages? 

I will close this issue, because it looks to me like the plugin is generally broken on your system, so let us resume the bug hunt in #3. If you really think that this is a completely different issues from #3, you can reopen it.

**Comment by zzottel** [comment]
This is solved, and was more a UI glitch of Conversations than a bug.

Just for the record, a clarification what exactly happened:

My resources: AG (Gajim) and AC (Conversations). The contact's only resource: BC (Conversations).

AC and BC were connected before AG was brought into the equation, and they were already talking to each other OMEMO-encrypted.

Now, when I added Gajim with the plugin, BC's Conversations showed the dialog to accept the new OMEMO key, but my own Conversations (AC) didn't. Thus, when BC wrote an encrypted message, I could see it in both in AC and AG. When I wrote an encrypted message with AC, however, it was not shown in AG.

When I started OMEMO-encrypting messages from AC to a completely unrelated contact in Conversations, I was asked to accept this contact's key as well as my own Gajim key. Since I did that, it works.

**Comment by kalkin** [comment]
> Now, when I added Gajim with the plugin, BC's Conversations showed the dialog to accept the new OMEMO key, but my own Conversations (AC) didn't. 

@iNPUTmice can you please have a look at this? I'm not sure if this is an issue with Conversations. I think i notice also that Conversations does not prompt you if you add a new own device.

**Comment by RalfJung** [comment]
It took us a while, but a friend and me just managed to get all messages visible on all devices, each of us using Conversation on a phone and Gajim on a PC (and of course Jabber servers that have Message Carbon enabled). We had to first send messages back and forth until we saw those red locks in Conversations, and then enable our "other" device by going to the account details. There was no automatic prompt, at least not for me. But that's a Cinversations issue. From all I can tell, the Gajim plugin mostly worked.
